plugins/org.eclipse.n4js.model/src/org/eclipse/n4js/AnnotationDefinition.java

```diff
@@ -258,6 +259,29 @@ public final class AnnotationDefinition {
 			.targets(FUNCTION_DECLARATION, N4_METHOD_DECLARATION)
 			.retention(RetentionPolicy.TYPE).end();
 
+	/**
+	 * Represents {@code export = x;} in .d.ts files.
+	 * <p>
+	 * Only to be used in .d.ts.
+	 */
+	public final static AnnotationDefinition EXPORT_EQUALS = define("ExportEquals")
+			.targets(SCRIPT)
+			.args(STRING_LITERAL)
+			.transitive()
+			.retention(RetentionPolicy.TYPE)
+			.end();
+
+	/**
+	 * States that the annotated {@link TClassifier classifier} owns an index signature on the .d.ts side.
+	 * <p>
+	 * Only to be used in .d.ts.
```

Comment:
Do we need a validation for that constraint? I.e. unsupported feature validation?

Reply:
Good idea, will do!